docs/data/data-grid/state/state.md

```diff
@@ -51,37 +52,37 @@ const pageSize = gridPageSizeSelector(
 
 {{"demo": "DirectSelector.js", "bg": "inline"}}
 
-### With `useGridSelector`
+### With useGridSelector
 
-If you want to access sole state value in the render of your components, you can use the `useGridSelector` hook.
+If you only need to access the state value in the render of your components, use the `useGridSelector` hook:
 
 ```tsx
 const pageSize = useGridSelector(apiRef, gridPageSizeSelector);
 ```
 
 :::warning
-This hook can only be used inside the context of the grid, such as custom components.
-Otherwise, you will have an error saying that the state is not initialized during the first render.
-:::warning
+This hook can only be used inside the context of the Data Grid, such as within custom components.
+Otherwise, you will get an error saying that the state is not initialized during the first render.
+:::
 
 {{"demo": "UseGridSelector.js", "bg": "inline"}}
 
 ### Catalog of selectors
 
-Some selectors are yet to be documented.
+Some selectors have not yet been documented.
 
 {{"demo": "SelectorsNoSnap.js", "bg": "inline", "hideToolbar": true}}
 
 ## Save and restore the state
 
-The current state of the grid can be exported using `apiRef.current.exportState()`.
+The current state of the Data Grid can be exported using `apiRef.current.exportState()`.
 It can then be restored by either passing the returned value to the `initialState` prop or to the `apiRef.current.restoreState()` method.
 
-Watch out for controlled models and their callbacks (`onFilterModelChange` if you use `filterModel`, for instance), as the grid will call those callbacks when restoring the state.
+Watch out for controlled models and their callbacks (`onFilterModelChange` if you use `filterModel`, for instance), as the Data Grid calls those callbacks when restoring the state.
 But if the callback is not defined or if calling it does not update the prop value, then the restored value will not be applied.
 
 :::warning
-To avoid breaking changes, the grid only saves the column visibility if you are using the [new api](/x/react-data-grid/column-visibility/#initialize-the-visible-columns)
+To avoid breaking changes, the Data Grid only saves the column visibility if you are using the [newer API](/x/react-data-grid/column-visibility/#initialize-the-visible-columns)
```

Comment:
Oh forgot to remove that one, we can remove this warning altogether, ti's outdated